### PR TITLE
[DOC] shorthand method and quotes/semicolons for serializers/json

### PR DESCRIPTION
**Pull request by knownasilya**

### CODE REVIEW

File: addon/serializers/json.js [hunk 1]
```diff
@@ -506,15 +506,17 @@ const JSONSerializer = Serializer.extend({
     import DS from 'ember-data';
 
     export default DS.JSONSerializer.extend({
-      normalize: function(typeClass, hash) {
+      normalize(typeClass, hash) {
         var fields = Ember.get(typeClass, 'fields');
+
```
Comment (locks) [inline addon/serializers/json.js]: lotsa trailing whitespace. github editor sucks, huh ;P

File: addon/serializers/json.js [hunk 2]
```diff
@@ -506,15 +506,17 @@ const JSONSerializer = Serializer.extend({
...
         var fields = Ember.get(typeClass, 'fields');
+        
         fields.forEach(function(field) {
           var payloadField = Ember.String.underscore(field);
           if (field === payloadField) { return; }
 
           hash[field] = hash[payloadField];
           delete hash[payloadField];
         });
+
```
Comment (locks) [inline addon/serializers/json.js]: whitespace

Comment (knownasilya) [inline addon/serializers/json.js]: Bah, github editor!

File: addon/serializers/json.js [hunk 3]
```diff
@@ -1248,15 +1249,16 @@ const JSONSerializer = Serializer.extend({
...
 
     export default DS.JSONSerializer.extend({
-      serializePolymorphicType: function(snapshot, json, relationship) {
-        var key = relationship.key,
-            belongsTo = snapshot.belongsTo(key);
-        key = this.keyForAttribute ? this.keyForAttribute(key, "serialize") : key;
+      serializePolymorphicType(snapshot, json, relationship) {
+        var key = relationship.key;
+        var belongsTo = snapshot.belongsTo(key);
+
```
Comment (locks) [inline addon/serializers/json.js]: whitespace

**Pull request merged**